src/isar/modules.py

```diff
@@ -177,26 +178,33 @@ def provide_mqtt_client(self, queues: Queues) -> MqttClientInterface:
         return None
 
 
+injector_modules: List[Module] = []
+module_config_keys: List[str] = []
```

Comment:
What is the reasoning behind moving these variables outside the function itself? This way they will always be in memory as "global" variables in this file while I'm not certain they need to be?

Reply:
It was so that the _append_key function would have access to them as if they where all inside a static class. 
The alternative is to pass them to each call to that function.
I'm not that comfortable with python yet, so if you think that's a better alternative then let's do that!

Reply:
@Christdej @tsundvoll 

Any thoughts on this? Personally something feels off to me about having them as open variables within the file in this manner as they are then exported and it's possible to import them. But I'm not certain if it's a bad or good pattern.

Reply:
I agree that it looks a bit off. It can be a symptom of the entire file being written a bit hard to read. I am looking into a better way to formulate this now.

Reply:
Maybe moving the functionality of this last part of the file to its own file could be a solution?

Reply:
It could be moved but then again it is very tightly coupled with the modules defined in this file.

I think I'll say approved from my side, but let @tsundvoll comment before you merge to see if he thought of a smooth solution.

Reply:
We found a smooth solution that resolves this comment.